### DISCUSSION TITLE
Unable to Add content on the collections

### DISCUSSION THREAD

**Discussion by sathish-ather** [outline/outline#3555] [Self Hosting]
Hey Guys,

Have deployed the outline app by using the docker-compose and able to create the topic, but not able to see any place to add the content on the topic.

PS : At the right side bottom, able to see some error msg "server connection lost - edits which you make will visible once you are online" --> Not sure which server, it tries to connect and why it shows connection lost..!

Could u please help us to fix the same.!

**Comment by sathish-ather** [comment]
Any update on this ?

**Reply by Limezy** [reply]
Hi, if you want some help we need some more details... 
What is showing up in the developer console ? Network tab ?
What is showing up in Outline logs ?
How does your .env look like ?
etc. etc...

**Comment by okontsograda** [comment]
I'm having this same issue and I'm seeing the following in my console log:

`WebSocket connection to 'wss://domain.com/collaboration/document.0e045537-ffb3-4d40-9aa7-689185aa7f9f' failed: `

I essentially am able to create new documents, add a title, but I'm unable to see the edit content section where I should be able to add text/images, etc.

**Comment by rtpacks** [comment]
Hey @sathish-ather @okontsograda, I have this same issue and I solved it. You can try to configure nginx's WebSocket, refer to the following configuration: 

```conf
server {
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $http_connection;

    location / {
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection $http_connection;
        proxy_http_version 1.1;
    }
}
```
